.github/workflows/stable-merge-check.yml

```diff
@@ -4,7 +4,8 @@ on:
   pull_request:
     branches:
       - stable
-    types: [opened, synchronize, reopened, edited]
+      - master
+    types: [opened, synchronize, reopened, edited, labeled]
```

Comment:
Adding labeled to the PR triggers and checking for it in the job condition won't help. When process-integration-command adds the label using GITHUB_TOKEN, GitHub doesn't trigger new workflow runs (to prevent infinite loops). The integration-test-check job won't run automatically.
Fix: After adding the label in process-integration-command, immediately run the check script in the same job: